internal/models/weather.model.go

```diff
@@ -1,10 +1,15 @@
 package models
 
-type WeatherData struct {
-	OpenWeather     FrontendWeatherResponse
-	TomorrowWeather FrontendWeatherResponse
+type WeatherInput struct {
+	City   string `json:"city" binding:"required"`
+	Source string `json:"source" binding:"required"`
 }
 
+// type WeatherData struct {
```

Comment:
If you don't need it better to take it out

Reply:
Fixed: Deleted this comment